CHANGELOG.md

```diff
@@ -385,7 +385,8 @@ Definitions:
 	*   Hisilicon HiKey Board, ARMv8-A (plat-hikey)
 *   AArch64 build of optee_os is now possible through the configuration `CFG_ARM64_core=y`
 *	Secure Storage: Data can be encrypted prior to their storage in the non-secure.
-	Build is configured using `CFG_ENC_FS=y`
+	Build is configured using `CFG_ENC_FS=y` (Note: this was later removed since
```

Comment:
Wasn't `CFG_ENC_FS` still around in the 2.2.0 tag? IIRC this was removed in the internal 2.2.0 -> the coming 2.3.0?
```
$ git lg -1 && git grep CFG_ENC_FS | wc -l
* c0c5d39 - (HEAD, tag: 2.2.0, tag: m/master) Update CHANGELOG.md for 2.2.0 (3 months ago) <Jerome Forissier>
34
```
I.e, the CHANGELOG.md shouldn't be changed, the history is still intact. What we should do is add a note about it in for the 2.3.0 updates.